db/sql/config.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+'use strict';
+
+const QueryFile = require('pg-promise').QueryFile;
+const schema = require('../../config.json').db.schema;
```

Comment:
So the config is actually parsed and processed during app startup. See this line..

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/app.js#L79

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/app.js#L94

And this `db` information is passed to `db/index.js` file. 

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/app.js#L433-L436

I would prefer that you extract or use schema config attribute from the config passed to db initializer, not directly from raw json file.

Reply:
The schema is a static piece of information. It is not needed anywhere except in the SQL loader, because it has no meaning outside of the SQL loader ;)

Reply:
I think best is to keep it in a single place where it is in fact used ;)

Reply:
We can override many config parameters from the command line, including the database name. Consider of the situation when later we add a new command line parameter to override the schema as well. In that case app will break. So when you are introducing  a new config param, use it from proper config, so later can be extended in command line parameters as well.